app/views/user_sessions/_form.html.slim

```diff
@@ -0,0 +1,17 @@
+.login_page
```

Comment:
just curious why partial? also login path is GET request and we need to either add another route (one for POST) or you can use user_session recource and do it like this: `form_for(:session, url: user_sessions_path)`